Pages/Product List/product-list.css

```diff
@@ -0,0 +1,99 @@
+.product__container{
+    display: flex;
+}
+.filter{
+    display: flex;
+    flex-direction: column;
+    width: 18rem;
+    min-height: 100vh;
+    border-radius: 1px solid grey;
+    padding: 1rem 2rem;
+    gap: 2rem;
+    border-right: 1px solid #111;
+}
+.filter__head,
+.filter__range{
+    display: flex;
+    justify-content: space-between;
+    gap: .8rem;
+}
+.filter__heading{
+    padding: .2rem 0; 
+}
+.slider{
+    width: 100%;
+    height: .438rem;
+    outline: none;
+    opacity: 0.7;
+}
+.filter__input{
+    padding: .3rem 0;
+}
+
+.product{
+   width: 80vw;
+   margin: 0 auto;
+   background-color: #f5f1f1;
```

Comment:
Use variables for colors if they are repeating

Reply:
Done